config/checker/com.intellij.astub

```diff
@@ -139,6 +139,8 @@ interface Application {
 @UIPackage package com.intellij.openapi.fileEditor;
 @UIPackage package com.intellij.openapi.fileTypes;
 
+@UI class SyntaxHighlighter {}
```

Comment:
Hmm syntax highlighting happens on UI thread in IntelliJ? :hushed: hmm that would severely reduce the responsiveness of the IDE if true :thinking:

Reply:
No, this was forced by making `com.intellij.openapi.fileTypes` `@UIPackage` unnecessarily.
Fixed.